addons/traefik/1.7.x/tests/00-install.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.crt /tmp/ca.crt
```

Comment:
these tests are assuming underlying kubernetes distro, to be kind. 
I would write the tests assuming they would be run in different distros like konvoy